### PR TITLE
 DBZ-3201 testsuite docker container

### PR DESCRIPTION
**Pull request by novotnyJiri**
https://issues.redhat.com/browse/DBZ-3201

### CODE REVIEW

File: jenkins-jobs/docker/debezium-testing-opneshift/testsuite-deployment.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+if [ "${DELETE_PROJECTS}" = false ] ;
+then exit 0;
+fi ;
+oc delete project ${OCP_PROJECT_DEBEZIUM} ; 
+oc delete project ${OCP_PROJECT_MYSQL} ; 
+oc delete project ${OCP_PROJECT_POSTGRESQL} ; 
+oc delete project ${OCP_PROJECT_SQLSERVER} ; 
+oc delete project ${OCP_PROJECT_MONGO} ; 
+oc delete project ${OCP_PROJECT_DB2} ; 
+echo finished
```
[the remark this answers is not in the document]
Comment (jcechace) [reply jenkins-jobs/docker/debezium-testing-opneshift/testsuite-deployment.sh:69]: I'd also like to see the file either segmented (put an empty line here and there ) or actually move things into functions to improve readability

**Comment by gunnarmorling** [comment]
Hey @jpechane, @novotnyJiri, what's needed for un-WIP-ing this?

**Review by jcechace — CHANGES_REQUESTED** [review]

File: jenkins-jobs/docker/debezium-testing-opneshift/testsuite-deployment.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+. /testsuite/library.sh
```
Comment (jcechace) [inline jenkins-jobs/docker/debezium-testing-opneshift/testsuite-deployment.sh:3]: prefer  ``source`` over execution  for example new environment variables would not to be propagated back to the original shell)

File: jenkins-jobs/docker/debezium-testing-opneshift/library.sh [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    OPTS=`getopt -o --long dbz-repository:,dbz-branch:,strimzi-repository:,strimzi-branch:,product-build,strimzi-downstream:,product-build:  -n 'clone-repositories' -- "$@"`
+    if [ $? != 0 ] ; then echo "Failed parsing options." >&2 ; exit 1 ; fi
+    eval set -- "$OPTS"
+
+    while true; do
+        case "$1" in
+            --dbz-repository )     DEBEZIUM_REPO=$2;               shift 2;;
+            --strimzi-repository ) STRIMZI_REPO=$2;                shift 2;;
+            --product-build )      PRODUCT_BUILD=$2;               shift 2;;
+            --strimzi_downstream ) STRIMZI_DOWNSTREAM_URL=$2;      shift 2;;
```
Comment (jcechace) [inline jenkins-jobs/docker/debezium-testing-opneshift/library.sh:37]: lets be consistent and use ``--strimzi-downstream``

**Review by jcechace — CHANGES_REQUESTED** [review]

**Review by jcechace — APPROVED** [review]
LGTM.

@jpechane  Can you have a quick look in case I missed something important?

**Comment by jpechane** [comment]
@novotnyJiri Just a cosmetic question - why the multiple empty lines at the end of files?

**Comment by novotnyJiri** [comment]
> @novotnyJiri Just a cosmetic question - why the multiple empty lines at the end of files?

I must have left them there when refactoring, will fix it right away.

**Pull request merged**